greynoise/pack.yaml

```diff
@@ -0,0 +1,18 @@
+---
+
+ref: greynoise
+name: GreyNoise
+description: >-
+    GreyNoise provides information on devices observed mass-scanning the internet.
+    This pack provides a set of actions to lookup IPs or query the GreyNoise API.
```

Comment:
I wish all pack descriptions were this clear about the services they integrate with - good job! 😄